### PR TITLE
Add last action (coinMarketCap and Nist)

### PR DESCRIPTION
**Pull request by Cleopha**
# Description

- Add Coin Market Cap Service check dominance of bitcoin (enter one parameters)
- Add Nist (check if new CVE are publish)

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [X] I have assigned this PR to myself
- [X] I have added at least 1 reviewer
- [ ] I have tested this code
- [X] I have added sufficient documentation in the code

### CODE REVIEW

File: obelix/idefix/services/coinmarketcap/client.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+		return fmt.Errorf("failed to parse body: %w", err)
+	}
+
+	old, err := dominance.GetRedisState(c.Operator.RC, taskID)
+	if err != nil {
+		return fmt.Errorf("failed to update redis state: %w", err)
+	}
+
+	err = dominance.LookForChange(c.Operator, taskID, old, owner, p.limit)
+	if err != nil {
```
Comment (0xpanoramix) [inline obelix/idefix/services/coinmarketcap/client.go]: You can inline this :)

File: obelix/idefix/services/coinmarketcap/domiance.go [hunk 2]
```diff
@@ -0,0 +1,126 @@
+package coinmarketcap
```
Comment (0xpanoramix) [inline obelix/idefix/services/coinmarketcap/domiance.go]: The filename contains a typo :/

File: obelix/idefix/services/coinmarketcap/domiance.go [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+	if err != nil {
+		return fmt.Errorf("failed to create new request: %w", err)
+	}
+
+	req.Header.Set("Accepts", "application/json")
+	req.Header.Add("X-CMC_PRO_API_KEY", ClientID)
+
+	resp, err := client.Do(req)
+	if err != nil {
+		return fmt.Errorf("failed tp send request: %w", err)
```
Comment (0xpanoramix) [inline obelix/idefix/services/coinmarketcap/domiance.go]: There's a typo here

File: obelix/idefix/services/nist/client.go [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+		return fmt.Errorf("failed to parse new cve data: %w", err)
+	}
+
+	// Retrieves the last state from the Redis DB.
+	old, err := cve.GetRedisState(c.Operator.RC, taskID)
+	if err != nil {
+		return fmt.Errorf("failed to update redis state: %w", err)
+	}
+
+	// Verifies if the event is in the range of < 10 minutes.
```
Comment (0xpanoramix) [inline obelix/idefix/services/nist/client.go]: This comment is inappropriate

**Review by 0xpanoramix — CHANGES_REQUESTED** [review]
LGTM :rocket:

File: obelix/idefix/services/coinmarketcap/client.go (hunk 1 above)
Comment (Cleopha) [reply obelix/idefix/services/coinmarketcap/client.go]: 0e41970fcd620eaa65b55297c75857ec39986eb5

File: obelix/idefix/services/coinmarketcap/domiance.go (hunk 2 above)
Comment (Cleopha) [reply obelix/idefix/services/coinmarketcap/domiance.go]: abe53828c9a08ec39de710174a918af0909572c0

File: obelix/idefix/services/coinmarketcap/domiance.go (hunk 3 above)
Comment (Cleopha) [reply obelix/idefix/services/coinmarketcap/domiance.go]: abe53828c9a08ec39de710174a918af0909572c0

File: obelix/idefix/services/nist/client.go (hunk 4 above)
Comment (Cleopha) [reply obelix/idefix/services/nist/client.go]: e2d71fa1ba536e4649f245006a17424acc4a0a2d

**Review by TomChv — APPROVED** [review]
LGTM

**Review by 0xpanoramix — APPROVED** [review]

**Pull request merged**